client/app/components/routes/operations/Operations.tsx

```diff
@@ -20,21 +20,22 @@ async function getOperations() {
     throw error;
   }
 }
-
 const formatTimestamp = (timestamp: string) => {
   if (!timestamp) return undefined;
 
   const date = new Date(timestamp).toLocaleString("en-CA", {
     month: "short",
     day: "numeric",
     year: "numeric",
+    // timeZone: "America/Vancouver",
```

Comment:
Hey is this a mistake? The only changes are some added commented code.

Reply:
Sorry, something went funky with the revert/rebase. Fixed now